### PR TITLE
HOCS-2995: update to React 17.0.2 initial broken commit

### PR DESCRIPTION
**Pull request by gary-newbolt**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -116,6 +113,7 @@
         }
     },
     "dependencies": {
+        "@babel/polyfill": "^7.12.1",
```
Comment (robertdeniszczyc2) [inline package.json]: This has been removed as a devDependency and the package is deprecated, should this be added here/could we change to the recommended alternative: https://babeljs.io/docs/en/babel-polyfill

**Review by robertdeniszczyc2 — COMMENTED** [review]
I've tried running the tests for this locally after a clean slate install with node 14.15.4 and get a whole bunch of failures, but can see the tests appear to have passed in the pipeline. Do the tests pass OK for you locally? Not sure why I'm seeing a difference.

**Comment by gary-newbolt** [comment]
> I've tried running the tests for this locally after a clean slate install with node 14.15.4 and get a whole bunch of failures, but can see the tests appear to have passed in the pipeline. Do the tests pass OK for you locally? Not sure why I'm seeing a difference.

**Comment by gary-newbolt** [comment]
> > I've tried running the tests for this locally after a clean slate install with node 14.15.4 and get a whole bunch of failures, but can see the tests appear to have passed in the pipeline. Do the tests pass OK for you locally? Not sure why I'm seeing a difference.

Hi Rob, I was running 14.17 as it happens however when I've switched to 14.15.4, my tests still pass. I did a clean-install as well to check, and they still pass. What exceptions are you getting?

**Comment by robertdeniszczyc2** [comment]
> > > I've tried running the tests for this locally after a clean slate install with node 14.15.4 and get a whole bunch of failures, but can see the tests appear to have passed in the pipeline. Do the tests pass OK for you locally? Not sure why I'm seeing a difference.
> 
> Hi Rob, I was running 14.17 as it happens however when I've switched to 14.15.4, my tests still pass. I did a clean-install as well to check, and they still pass. What exceptions are you getting?

Apologies looks like I didn't have all the changes on the branch when I checked it out. Run and all passed ok now

File: package.json (hunk 1 above)
Comment (gary-newbolt) [reply package.json]: I can't quite remember why I brought it in to begin with. I've removed it and rerun the tests / rebuilt the project and had a brief play with the app. All seems to work with this removed.

**Review by robertdeniszczyc2 — APPROVED** [review]

**Pull request merged**